src/components/spaces/show/Toolbar/import_from_slurp_form.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react'
+
+import ReactDOM from 'react-dom'
+import Icon from 'react-fa'
+
+import e from 'gEngine/engine'
+
+import * as elev from 'server/elev/index'
+
+import './style.css'
+
+export class ImportFromSlurpForm extends Component {
+  state = {
+    value: ""
+  }
+
+  componentDidMount() {
+    // TODO(Ozzie): Ozzie, any idea how to get this to focus? I can't seem to...
+    this.refs.textarea.focus()
+    ReactDOM.findDOMNode(this.refs.textarea).focus()
```

Comment:
Can this just be removed?  It doesn't matter much, and I don't want it here if it is not working.